extensions/3rdparty/LyricWiki/server.php

```diff
@@ -2477,13 +2476,19 @@ function requestStarted($funcName, $requestData){
 	 */
 	if( !wfReadOnly() ) {
 		if(defined('TRACK_REQUEST_RUNTIMES') && TRACK_REQUEST_RUNTIMES) {
-			$db = lw_connect();
 			$requestData = str_replace("'", "[&apos;]", $requestData);
-			$queryString = "INSERT INTO apiRequests (requestedThrough, requestedFunction, requestData, requestTime)";
-			$queryString.= " VALUES ('$REQUEST_TYPE', '".mysql_real_escape_string($funcName, $db)."', '".mysql_real_escape_string($requestData, $db)."', NOW())";
-			if( mysql_query($queryString, $db ) ){
-				$retVal = mysql_insert_id( $db );
-			}
+			
+			$dbr = wfGetDB( DB_MASTER );
+			$dbr->insert(
+				'apiRequests',
+				[
+					"requestedThrough" => $REQUEST_TYPE,
+					"requestedFunction" => $dbr->addQuotes(funcName),
+					"requestData" => $dbr->addQuotes(requestData),
+					"requestTime" => "NOW()",
+				],
+				__METHOD__
+			);
 		}
 	}
 	return $retVal;
```

Comment:
Nope, that's definitely wrong. Needed to add ->insertId().  Also noticed that it was called "dbr" here instead of "dbw" so fixed both of those issues.